### PR TITLE
fixed undefined question length const

### PR DESCRIPTION
**Pull request by alexey-petrov**
fixed undefined question length const
Occurs after registration on venues with questions ( possibly )

### CODE REVIEW

File: src/pages/Account/Questions.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const Questions: React.FC = () => {
     return <LoadingPage />;
   }
 
-  const numberOfQuestions = sovereignVenue?.profile_questions.length;
+  const numberOfQuestions = sovereignVenue?.profile_questions?.length;
```
Comment (sunny-viktoryia) [inline src/pages/Account/Questions.tsx:102]:
```suggestion
  const numberOfQuestions = sovereignVenue?.profile_questions?.length ?? 0;
```

Comment (alexey-petrov) [reply src/pages/Account/Questions.tsx]: fixed 58ada0b

File: src/pages/Account/CodeOfConduct.tsx [hunk 2]
```diff
@@ -141,7 +141,7 @@ export const CodeOfConduct: React.FC = () => {
 
   const codeOfConductQuestions = IS_BURN
     ? BURN_CODE_OF_CONDUCT_QUESTIONS
-    : venue.code_of_conduct_questions;
+    : venue?.code_of_conduct_questions || [];
```
Comment (sunny-viktoryia) [inline src/pages/Account/CodeOfConduct.tsx:144]:
```suggestion
    : venue?.code_of_conduct_questions ?? [];
```

**Review by sunny-viktoryia — APPROVED** [review]
Looks good to me. 🙃

**Pull request merged**